Clarifications of did:tdw features (abstract) and definitions - V2 (#85)
bcgov/trustdidweb: spec/definitions.md

[[def: DID Portability, DID:tdw portability, `DID:tdw` portability]]

~ `did:tdw` [[ref: portability]] encompasses the ability to change the DID string for the
DID while retaining the [[ref: SCID]] and the history of the DID. This is useful
when forced to change (such as when an organization is acquired by another,
resulting in a change of domain names) and when changing DID hosting service
providers.

[[def: did:web]]

> ~ `DID:web` as described in the [W3C specification](https://w3c-ccg.github.io/did-method-web/)

Reviewer: ```suggestion
~ `did:web` as described in the [W3C specification](https://w3c-ccg.github.io/did-method-web/)
```

is a DID method that leverages the Domain Name System (DNS) to perform the DID operations.
It is valued for its simplicity and ease of deployment compared to DID methods that are
based on distributed ledgers or blockchain technology, but also comes with increased

> challenges related to trust and security. DID:web provides a starting point for `DID:tdw`,

Reviewer: ```suggestion
challenges related to trust and security. `did:web` provides a starting point for `did:tdw`,
```

> which complements did:web with specific features to address the limitation of `DID:web`

Reviewer: ```suggestion
which complements `did:web` with specific features to address the limitations
```
